src/Options.js

```diff
@@ -322,6 +322,13 @@ export class Options {
         this.setOption(name, object[name]);
     });
     this.modules = object.modules || this.modules;
+    if (!!object.atscript) {
+      this.atscript = !!object.atscript;      
+    }
+    if (!!object.experimental) {
```

Comment:
This is problematic. atscript sets a few other options to true so this might override other options. For example:

```js
setFromObject({atscript: true, typeAssertions: false})
```

would ignore the second property.